src/transformers/models/mt5/modeling_mt5.py

```diff
@@ -818,10 +819,10 @@ def _shift_right(self, input_ids):
         decoder_start_token_id = self.config.decoder_start_token_id
         pad_token_id = self.config.pad_token_id
 
-        assert decoder_start_token_id is not None, (
-            "self.model.config.decoder_start_token_id has to be defined. In MT5 it is usually set to the pad_token_id."
-            " See MT5 docs for more information"
-        )
+        if decoder_start_token_id is None:
+            raise ValueError(
+                "self.model.config.decoder_start_token_id has to be defined. In MT5 it is usually set to the pad_token_id. See MT5 docs for more information."
```

Comment:
Same here re line length
```suggestion
                "self.model.config.decoder_start_token_id has to be defined. In MT5 it is usually set to the pad_token_id."
                " See MT5 docs for more information."
```